docs/content.zh/docs/learn-flink/etl.md

```diff
@@ -223,11 +215,11 @@ minutesByStartCell
 
 在 Flink 不参与管理状态的情况下，你的应用也可以使用状态，但 Flink 为其管理状态提供了一些引人注目的特性：
 
-* **本地性**: Flink 状态是存储在使用它的机器本地的，并且可以以内存访问速度来获取
-* **持久性**: Flink 状态是容错的，例如，它可以自动按一定的时间间隔产生 checkpoint，并且在任务失败后进行恢复
-* **纵向可扩展性**: Flink 状态可以存储在集成的 RocksDB 实例中，这种方式下可以通过增加本地磁盘来扩展空间
-* **横向可扩展性**: Flink 状态可以随着集群的扩缩容重新分布
-* **可查询性**: Flink 状态可以通过使用 [状态查询 API]({{< ref "docs/dev/datastream/fault-tolerance/queryable_state" >}}) 从外部进行查询。
+- **本地性**: Flink 状态是存储在使用它的机器本地的，并且可以以内存访问速度来获取
```

Comment:
why should we change this?

Reply:
Well, it is my Markdown with prettier plugin to auto-format it. I know it is unnecessary, because either `-` or `*` makes a list item in Markdown. By the way, I think it would be nicer to deploy a standard Markdown formatter.

Reply:
I added an extra [jira](https://issues.apache.org/jira/browse/FLINK-31177) in terms of markdown formatting.